src/flaskapp/api.py

```diff
@@ -0,0 +1,226 @@
+from flask import Flask, request
+
+from src.users.user_profile import (
+    get_user_profile,
+    get_user_profiles,
+    create_user_profile,
+    update_user_profile,
+)
+
+from src.teams.team_profile import (
+    get_team_profile,
+    get_team_profiles,
+    create_team_profile,
+    update_team_profile,
+)
+from src.teams.team_complete import team_complete
+from src.teams.user_leave import user_leave
+
+from src.teams.unify.team_invite import team_invite
+from src.teams.unify.team_confirm import team_confirm
+from src.teams.unify.team_rescind import team_rescind
+from src.teams.unify.team_reject import team_reject
+
+# from src.matching.team_recommendations import get_team_recommendations
+
+from src.flaskapp.util import format_string
+from src.flaskapp.auth import authenticate
+
+app = Flask(__name__)
+
+@app.route("/", methods=["GET"])
+def index():
+    return {"message": "Welcome to TeamRU!"}, 200
+
+############################## USERS ##############################
+
+
+@app.route("/users", methods=["GET", "POST"])
+@authenticate
+def users(email):
+    email = None  # unused
+
+    if request.method == "GET":
+        # Filter response using query parameters
```

Comment:
Do we ever need to filter on the users? Just teams right.

Reply:
Yup for now there should be no reason to pull all users. I think we'll still keep it in there (might be useful for organizers) and require permissions/additional search params later if needed, but definitely not a priority.